cmd/guard.go

```diff
@@ -0,0 +1,83 @@
+package cmd
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"strings"
+
+	"github.com/hashicorp/go-version"
+	"github.com/leancodepl/poe2arb/flutter"
+	"github.com/spf13/cobra"
+)
+
+type flutterConfigKey int
+
+const key flutterConfigKey = 1
+
+func contextWithFlutterConfig(ctx context.Context, flutterConfig *flutter.FlutterConfig) context.Context {
+	return context.WithValue(ctx, key, flutterConfig)
+}
+
+func flutterConfigFromContext(ctx context.Context) (*flutter.FlutterConfig, bool) {
```

Comment:
`getFlutterConfig` (just like `getLogger`)

Reply:
`getFlutterConfig` function is already defined in this scope. We can change existing `getFlutterConfig` to `getFlutterCfg` and `flutterConfigFromCommand` to `getFlutterConfig`, but IMO it won't be idiomatic.